Scripts/script-DemistoLeaveAllInvestigations.yml

```
@@ -0,0 +1,52 @@
+commonfields:
+  id: DemistoLeaveAllInvestigations
+  version: -1
+name: DemistoLeaveAllInvestigations
+script: |
+  function addEntry(text) {
+      entry =  [{ContentsFormat: formats.markdown, Type: entryTypes.note, Contents: text}];
+      executeCommand("addEntries", {"entries":JSON.stringify(entry)});
+  }
+
+  var account = 'acc_' + incidents[0].account;
```

acc_ is only for multi tenant. in non multi tenant this should be simply empty

```suggestion
  var account = incidents[0].account ?  'acc_' + incidents[0].account : "";
```